tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/HasBeenUpdatedCounter.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.tritium.metrics.registry;
+
+import com.codahale.metrics.Counter;
+
+public final class HasBeenUpdatedCounter extends Counter {
```

Comment:
very open to better names here - I also considered just calling it `com.palantir.tritium.metrics.registry.Counter`, or `Counter2` or `LazyCounter` or `UsageTrackingCounter`